src/main/resources/align-apache-kafka.json

```diff
@@ -0,0 +1,18 @@
+{
+    "align": [
+        {
+            "name": "align-apache-kafka",
```

Comment:
this one looks good 

Any chance you could add an integration test similar to https://github.com/nebula-plugins/gradle-resolution-rules/blob/main/src/integTest/groovy/nebula/plugin/resolutionrules/AlignSpringKafkaIntegrationSpec.groovy ?